pkg/gofr/datasource/surrealdb/utils.go

```diff
@@ -0,0 +1,76 @@
+package surrealdb
+
+import (
+	"fmt"
+	"io"
+	"regexp"
+	"strings"
+
+	"go.opentelemetry.io/otel/trace"
+)
+
+var rgx = regexp.MustCompile(`\s+`)
+
+// Clean takes a string query as input and performs two operations to clean it up:
+// 1. It replaces multiple consecutive whitespace characters with a single space.
+// 2. It trims leading and trailing whitespace from the string.
+// The cleaned-up query string is then returned.
+func clean(query string) string {
```

Comment:
```suggestion
// clean takes a string query as input and performs two operations to clean it up:
// 1. It replaces multiple consecutive whitespace characters with a single space.
// 2. It trims leading and trailing whitespace from the string.
// The cleaned-up query string is then returned.
func clean(query string) string {
```


I'm unsure why this one wasn't reported

Reply:
I think the reason is here
https://github.com/gofr-dev/gofr/pull/1426

The go.mod separations might cause this